Assets/LeapMotionModules/InteractionEngine/Scripts/InteractionManager.cs

```diff
@@ -118,55 +107,29 @@ public partial class InteractionManager : MonoBehaviour {
       }
     }
 
-    protected INTERACTION_SCENE _scene;
     private bool _hasSceneBeenCreated = false;
-    private bool _enableGraspingLast = false;
-
-    protected ActivityManager _activityManager = new ActivityManager();
-    protected ShapeDescriptionPool _shapeDescriptionPool;
-
-    //Maps the Interaction instance handle to the behaviour
-    //A mapping only exists if a shape instance has been created
-    protected Dictionary<INTERACTION_SHAPE_INSTANCE_HANDLE, IInteractionBehaviour> _instanceHandleToBehaviour = new Dictionary<INTERACTION_SHAPE_INSTANCE_HANDLE, IInteractionBehaviour>();
 
+    public ActivityManager _activityManager = new ActivityManager();
     protected Dictionary<int, InteractionHand> _idToInteractionHand = new Dictionary<int, InteractionHand>();
     protected List<IInteractionBehaviour> _graspedBehaviours = new List<IInteractionBehaviour>();
+    protected HeuristicGrabClassifier _grabClassifier;
+    [NonSerialized]
+    public List<PhysicsUtility.SoftContact> softContacts = new List<PhysicsUtility.SoftContact>(80);
```

Comment:
When I added soft contact to my refactor, I added did this:
```
/// <summary> Stores data for implementing Soft Contact for InteractionHands. </summary>
    [NonSerialized]
    public List<PhysicsUtility.SoftContact> _softContacts = new List<PhysicsUtility.SoftContact>(80);
    /// <summary> Stores data for implementing Soft Contact for InteractionHands. </summary>
    [NonSerialized]
    public Dictionary<Rigidbody, PhysicsUtility.Velocities> _softContactOriginalVelocities = new Dictionary<Rigidbody, PhysicsUtility.Velocities>(5);
```
I thought those comments were sufficient to demonstrate that if you don't know how soft contact works, you probably shouldn't be messing with them